var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -115,14 +131,23 @@ def install(self, spec, prefix):
                 'METIS_OPT=-I%s' % spec['metis'].prefix.include,
                 'METIS_LIB=%s' % metis_lib])
 
-        if '+mpi' in spec:
+        if 'mpi' in spec:
```

Comment:
Perhaps an easier way to remember this is that `foo in spec` is the same as `spec.satisifies('foo')`.